test/integration/targets/openssl_csr/aliases

```diff
@@ -0,0 +1,2 @@
+posix/ci/group1
+destructive
```

Comment:
You can remove `destructive` since there doesn't appear to be anything destructive about these tests.

The same is true for the other tests.